### PR TITLE
"adding files for lesson 14"

### PR DESCRIPTION
**Pull request by MihaelaGit**

### CODE REVIEW

File: lesson_14/lesson_14_mihaela_vaida/Challenge.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Lesson 14 - Operator Overloading
+
+## Challenge:
+
+
+Design a movie class with 2 more derivative classes: ArtisticMovies and Series!
+
+Create a challenge similar to the ones we included in the previous lessons. The challenge must reach the following points:
```
Comment (Liitle) [inline lesson_14/lesson_14_mihaela_vaida/Challenge.md]: you can remove the lines that are not related to the challenge :smile:

Comment (MihaelaGit) [reply lesson_14/lesson_14_mihaela_vaida/Challenge.md]: done!

**Review by Liitle — APPROVED** [review]
